### PR TITLE
chore: visual regression files cleanup

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

When we release css updates we need a visual regression test env

## 🚀 Changes
 
- Add exclusions glob for lerna
- Cleaned up bootstrap files

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

**Review by helendurranttw — APPROVED** [review]

File: packages/components/.storybook/config.js [hunk 1]
```diff
@@ -54,4 +54,9 @@ addParameters({
   },
 });
 
-configure(require.context('../src', true, /\.story\.js$/), module);
+const requires = [require.context('../src', true, /\.story\.js$/)];
```
Comment (andreapiras) [inline packages/components/.storybook/config.js:57]: include only if in percy mode

File: packages/components/.storybook/config.js [hunk 2]
```diff
@@ -54,4 +54,9 @@ addParameters({
...
+if (process.env.NODE_ENV === 'percy') {
```
Comment (andreapiras) [inline packages/components/.storybook/config.js:58]: include only if in percy mode

File: packages/components/test/neptuneCssTest/NeptuneCssTest.story.js [hunk 3]
```diff
@@ -9,7 +9,10 @@ export default {
 
 export const basic = () => (
   <>
-    <Alert type="warning">Please use only for visual regression testing</Alert>
+    <Alert type="error">
+      This page is used only for visual regression testing. We DO NOT garantee support for the
```
Comment (helendurranttw) [inline packages/components/test/neptuneCssTest/NeptuneCssTest.story.js:13]: Typo: guarantee

**Comment by helendurranttw** [comment]
This looks much cleaner. I think you should add a note to CONTRIBUTING.md in components explaining how people can run these tests locally, seeing as I don't think we'll have Percy on CI any time soon.

**Comment by helendurranttw** [comment]
Also please update the PR with a better description / title :)

**Review by helendurranttw — APPROVED** [review]

**Pull request merged**